app/tests/spec/models/password_strength/password_strength_balloon.js

```diff
@@ -0,0 +1,188 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { assert } from 'chai';
+import AuthErrors from 'lib/auth-errors';
+import PasswordModel from 'models/password_strength/password_strength_balloon';
+
+let model;
+
+describe('models/password_strength/password_strength_balloon', () => {
+  before(() => {
+    model = new PasswordModel({
+      email: 'testuser@testuser.com'
```

Comment:
A quick observation from local testing (not a real issue), tried `a@mozilla.com` and it wouldn't let me create any password with the letter `a`. This is working as intended since it is part of the email, however I can see some users with custom domains and small emails getting caught no matter how long their password is.

Reply:
>  I can see some users with custom domains and small emails getting caught no matter how long their password is.

This is something I hadn't considered. Maybe there should be a lower bound on the local length, like 4 characters?

Reply:
@shane-tomlinson Adding a lower bound of 4 characters would definitely help this. Your call!